armi/physics/neutronics/settings.py

```diff
@@ -125,8 +130,8 @@ def defineSettings():
             label="Multigroup Cross Sections Generation",
             description="Generate multigroup cross sections for the selected particle "
             "type(s) using the specified lattice physics kernel (see Lattice Physics "
-            "tab). When not set, the XS library will be auto-loaded from an existing ISOTXS "
-            "within then working directory and fail if the ISOTXS does not exist.",
+            "tab). When not set, the XS library will be auto-loaded from an existing "
+            "ISOTXS in the working directory, but fail fail if there is no ISOTXS.",
```

Comment:
```suggestion
            "ISOTXS in the working directory, but fail if there is no ISOTXS.",
```
no need to say fail fail?